<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_newWindow</class>
 <widget class="QDialog" name="Edit_newWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EDIT_ADDRESS</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_Info">
       <property name="text">
        <string>INFO</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLineEdit" name="lineEdit_SearchName"/>
     </item>
     <item row="8" column="3">
      <widget class="QPushButton" name="pushButton_Enter">
       <property name="text">
        <string>Enter</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="pushButton_Edit">
       <property name="text">
        <string>EDIT!</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_SearchName">
       <property name="text">
        <string>Search
Name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="lineEdit_Sex"/>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="lineEdit_Name"/>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="lineEdit_Num"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_Number">
       <property name="text">
        <string>Number</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_Name">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_Sex">
       <property name="text">
        <string>Sex(M/W)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_Info"/>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="lineEdit_Address"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_Address">
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
